src/sentry/utils/snuba.py

```diff
@@ -74,7 +74,10 @@ def query(start, end, groupby, conditions=None, filter_keys=None,
         if col in snuba_map:
             keys = [snuba_map[col][k] for k in keys if k in snuba_map[col]]
         if keys:
-            conditions.append((col, 'IN', keys))
+            if len(keys) == 1 and keys[0] is None:
+                conditions.append((col, 'IS NULL', None))
+            else:
+                conditions.append((col, 'IN', keys))
```

Comment:
Out of curiosity, any idea what happens here if a `None` value ends up in `keys`?

Reply:
Yes, so weird Clickhouse behavior, `IN (Null)` or `= Null` just doesn't match anything.

So `col IN (Null)` doesn't match any rows. but `col IN (Null, 1)` will match the ones with `col = 1` at least

Reply:
I wonder if we should consider adding an assertion that `keys` doesn't contain `None` to avoid burning ourselves later? Seems pretty unlikely though so maybe overkill